docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -450,3 +450,31 @@ When building an Uber-Jar you can specify entries that you want to exclude from
 <1> The entries are relative to the root of the generated Uber-Jar. You can specify multiple entries by adding extra `<ignoredEntry>` elements.
 
 
+Uber-Jar's name can be configurable via a `finalName` configuration option.
+
+[source,xml,subs=attributes+]
+----
+<build>
+    <plugins>
+        <plugin>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-maven-plugin</artifactId>
+            <version>${quarkus.version}</version>
+            <configuration>
+                <uberJar>true</uberJar>
+                <finalName>uber-jar-name</finalName> <1>
```

Comment:
We should also support the Maven `finalName`. See https://maven.apache.org/pom.html.

Reply:
@cescoffier I’ll update the doc about this too. Will it (the two ways to configure finalName) lead to confusions?

Reply:
@aloubyansky WDYT about this?

Reply:
I think we should change the name of the config option to reflect what it is about. As it is not the *final* name, at least, atm.